test/integration/extensionRequests.test.js

```diff
@@ -362,7 +362,7 @@ describe("Extension Requests", function () {
           const extensionRequestId1 = res.body.extensionRequest.id;
           chai
             .request(app)
-            .get(`/logs/extensionRequest/?body.assignee=${user.id}&meta.taskId=${taskId2}`)
+            .get(`/logs/extensionRequests/?body.assignee=${user.id}&meta.taskId=${taskId2}`)
```

Comment:
Hey @ivinayakg just a doubt is this a route?, i don't see it [here](https://github.com/Real-Dev-Squad/website-api-contracts/tree/main/logs#requests) and also why not it's like below? what's the convention?
```javascript
/logs/extension-requests
```

Reply:
yeah because here we are trying to access the model